docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,231 @@
+
+Every PR should have a related issue that it is fixing. If you are going to be making a 
+change, make an issue for it and assign yourself to be the one to fix it.
+
+Be sure to observe and follow conventions for how Issues should be named and described and how to ping other people for input.
+ 
+Our team tends to self-assign issues. If you would like to take on a non-trivial issue, please ask us for input early and often.
+
+Don’t just follow the template blindly. For example, remove the “Brief summary of issue” header and put your summary of the issue there.
+Don’t leave obvious template words like “(if applicable)” in your issue.
+        
+## 3. Pull Requests
+Once you have created a new branch and pushed an initial commit to it, you will see the option to make a pull request on GitHub. 
+Make sure that the branch you are merging into is the branch that you checked-out when you first started making changes, such as main or a release-specific branch.
+
+PRs must be reviewed according to this [checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist), and must be approved by two reviewers before they are merged. Don’t forget to think about documentation updates like the TestPlan, changelog, and developer guide updates -- these must accompany your PR.
+
+Make sure to note special rules for SQL patches. Note that changes to Raisinbread data do not need patches, because RaisinBread is maintained by sourcing the project patches.
+
+**Important:** Do not merge your own PR or any other PR. After a PR has been reviewed and approved by a senior member of the LORIS team, it will be merged eventually. If your PR is on a release branch and has been ready to merge for a few days, you can add a comment that it is ready to merge. 
+
+It is your responsibility to be on top of your PRs and any Issues that need follow-up.  This includes answering questions and comments, acknowledging and making change requests, and requesting re-review after you’ve addressed someone’s change requests. 
+This is important for the developer team’s productivity -- it prevents PRs from lagging and prevents confusion as to why a PR has stalled or if you’ve seen someone’s comments.
+When issuing a PR, you can request a specific reviewer (this is not the same as assigning a PR to someone) if you know you would like someone in particular to review a PR. 
+This is especially important in Release-Test/Fix cycles when we want PRs to be reviewed and resolved quickly.
+
+### PR Formatting
+Be sure to observe and follow team conventions for how PRs should be named and described. 
+
+**Title:** [module_name] Quick description   
+
+**[Contents](../../../.github/PULL_REQUEST_TEMPLATE.md) of the initial comment:** Include a brief summary of the changes and rationale, links to related issues or PRs, and testing instructions if there are any. 
```

Comment:
Absolute link is required here : https://github.com/AlexandraLivadas/Loris/blob/master/.github/PULL_REQUEST_TEMPLATE.md, ottherwise RTD will throw a 404 (this .md is not included in RTD).